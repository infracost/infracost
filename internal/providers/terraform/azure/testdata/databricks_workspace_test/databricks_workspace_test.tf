provider "azurerm" {
  features {}
  skip_provider_registration = true
}

resource "azurerm_resource_group" "example" {
  name     = "example-resources"
  location = "eastus"
}

resource "azurerm_databricks_workspace" "standard" {
  name                = "databricks-test"
  resource_group_name = azurerm_resource_group.example.name
  location            = azurerm_resource_group.example.location
  sku                 = "standard"
}

resource "azurerm_databricks_workspace" "premium" {
  name                = "databricks-test"
  resource_group_name = azurerm_resource_group.example.name
  location            = azurerm_resource_group.example.location
  sku                 = "premium"
}

resource "azurerm_databricks_workspace" "trial" {
  name                = "databricks-test"
  resource_group_name = azurerm_resource_group.example.name
  location            = azurerm_resource_group.example.location
  sku                 = "trial"
}

resource "azurerm_databricks_workspace" "non_usage" {
  name                = "databricks-test"
  resource_group_name = azurerm_resource_group.example.name
  location            = azurerm_resource_group.example.location
  sku                 = "standard"
}
