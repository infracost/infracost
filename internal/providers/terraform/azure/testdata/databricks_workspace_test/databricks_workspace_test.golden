
 Name                                         Monthly Qty  Unit                  Monthly Cost 
                                                                                              
 azurerm_databricks_workspace.non_usage                                                       
 ├─ All-purpose compute DBUs             Monthly cost depends on usage: $0.40 per DBU-hours   
 ├─ Jobs compute DBUs                    Monthly cost depends on usage: $0.15 per DBU-hours   
 └─ Jobs light compute DBUs              Monthly cost depends on usage: $0.07 per DBU-hours   
                                                                                              
 azurerm_databricks_workspace.premium                                                         
 ├─ All-purpose compute DBUs                          500  DBU-hours                  $275.00 
 ├─ Jobs compute DBUs                               1,000  DBU-hours                  $300.00 
 └─ Jobs light compute DBUs                         2,000  DBU-hours                  $440.00 
                                                                                              
 azurerm_databricks_workspace.standard                                                        
 ├─ All-purpose compute DBUs                        1,000  DBU-hours                  $400.00 
 ├─ Jobs compute DBUs                               2,000  DBU-hours                  $300.00 
 └─ Jobs light compute DBUs                         4,000  DBU-hours                  $280.00 
                                                                                              
 OVERALL TOTAL                                                                      $1,995.00 
──────────────────────────────────
5 cloud resources were detected:
∙ 3 were estimated, all of which include usage-based costs, see https://infracost.io/usage-file
∙ 2 were free:
  ∙ 1 x azurerm_databricks_workspace
  ∙ 1 x azurerm_resource_group